day_7/10_speckled_frogs3.rb

```diff
@@ -0,0 +1,12 @@
+p "Input number of froggy friends:"
+frog = gets.chomp.to_i
+
+while frog > 1
+  p "#{frog} speckled frogs sat on a log \n eating some most delicious bugs. \n One jumped in the pool where it's nice and cool,\n then there were #{frog - 1} speckled frogs. \n"
+  frog -= 1
+end
```

Comment:
Nice use of iteration for the rhyme. Also good catch on plurality for the last case. FYI `p` is not rendering the `\n` whereas `puts` will.